pages/en/lb4/Getting-started.md

```diff
@@ -46,12 +54,60 @@ const app = new Application({
   await app.start();
   console.log(`REST server listening on port ${server.getSync('rest.port')}`);
 })();
+
+```
+
+Next, create a file called tsconfig.json to indicate that this is a TypeScript project and specify the TypeScript compiler options required.  For more details, see [TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+{% include code-caption.html content="tsconfig.json" %}
+```
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "es2017",                          
+    "module": "commonjs",                     
+
+    /* Experimental Options */
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true
+  }
+}
+```
+
+Finally, create a file called package.json. 
+
+{% include code-caption.html content="package.json" %}
+```
+{
+  "name": "getting-started",
+  "description": "Getting Started for LoopBack 4",
+  "authors": "you",
+  "version": "1.0.0",
+  "scripts": {
+    "start": "tsc && node index.js"
+  }
+}
+```
+
+### Running the application
+
+To install the required packages, enter:
+```
+npm install --save @loopback/core @loopback/rest @types/node typescript
```

Comment:
Would it be good if we put dependencies in `package.json` file instead of appending in cmd?

Reply:
I believe `--save` should write it in `package.json`, but yea we could pre-populate `package.json`. If they do it this way, though, they get the latest versions to test with (which could be a bad thing if there are breaking changes).

Reply:
Please note `@types/node typescript` should be devDependencies. They can be added to `package.json` using `--save-dev`.

Reply:
Would it work better to put everything in `package.json`, so that the user just need to run `npm i` as @jannyHou suggested in the first comment? 
```
"dependencies": {
    "@loopback/core": "^4.0.0-alpha.16",
    "@loopback/rest": "^4.0.0-alpha.3"
  },
  "devDependencies": {
    "@types/node": "^8.0.34",
    "typescript": "^2.5.3"
  }
```

Reply:
`typescript` is a regular (production) dependency, because we are running `tsc` as part of `npm start`. Not a big deal, since this is just an example and nobody is going to use this TS setup in a real project (at least I hope so!)